cluster/addons/cloud-controller-manager/cloud-node-controller-role.yaml

```diff
@@ -154,4 +160,4 @@ rules:
   - pods/status
   verbs:
   - list
-  - delete
+  - delete
```

Comment:
Let's be more care with these straw edits, might pollute history.

Reply:
Updated